kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgent.java

```diff
@@ -24,9 +25,13 @@
  * The presence of this file is tested via a Kube "exec" readiness probe to determine when the broker is ready.
  */
 public class KafkaAgent {
-
     private static final Logger LOGGER = LoggerFactory.getLogger(KafkaAgent.class);
 
+    // KafkaYammerMetrics class in Kafka 3.3+
+    private static final String YAMMER_METRICS_IN_KAFKA_3_3_AND_LATER = "org.apache.kafka.server.metrics.KafkaYammerMetrics";
+    // KafkaYammerMetrics class in Kafka 3.2-
+    private static final String YAMMER_METRICS_IN_KAFKA_3_2_AND_EARLIER = "kafka.metrics.KafkaYammerMetrics";
```

Comment:
just curious, is that normal it doesn't have a domain prefix like `org.apache`?

Reply:
That is how the whole Kafka `core` code written in Scala uses as a package. So not much we can do about it.